syntax = "proto3";

package production;

// ---------- SERVICE ----------

service ProductionService {
  rpc GetProduction (GetProductionRequest)
      returns (GetProductionResponse);
}

// ---------- REQUEST ----------

message GetProductionRequest {
  string source_id = 1;
}

// ---------- RESPONSE ----------

message GetProductionResponse {
  string source_id = 1;
  double value = 2;
  int64 timestamp = 3;
}
